rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/RocketChatWidgets.java

```diff
@@ -0,0 +1,26 @@
+package chat.rocket.android.widget;
+
+import android.content.Context;
+import com.facebook.drawee.backends.pipeline.DraweeConfig;
+import com.facebook.drawee.backends.pipeline.Fresco;
+import com.facebook.imagepipeline.backends.okhttp3.OkHttpImagePipelineConfigFactory;
+import com.facebook.imagepipeline.core.ImagePipelineConfig;
+
+import chat.rocket.android.widget.fresco.ImageFormatConfigurator;
+import okhttp3.OkHttpClient;
+
+public class RocketChatWidgets {
```

Comment:
Hmm, I've changed my mind a little... Leave this classname as it is, for now.

Also, https://github.com/RocketChat/Rocket.Chat.Android/pull/188/commits/32aceba380c2c5b9de1ed1f86bb481fa3dfceed7 looks better :)

Sorry for confusing, and thank you for discussion.